tests/MoBi.Tests/Presentation/SelectReferencePresenterSpecs.cs

```diff
@@ -229,4 +232,88 @@ public void should_return_an_empty_list()
          _result.ShouldBeEmpty();
       }
    }
+
+   public abstract class concern_for_SelectReferencePresenter_with_node_text : ContextSpecification<ISelectReferencePresenter>
```

Comment:
typically we derive from concern_for_SelectReferencePresenter
Why aren't you doing this here? you would inherit most of the fake already?

Reply:
in fact, I started doing that, but, I need some things which are different. 
As this actual implementations instead of the fakes:
   _objectBaseDTOMapper = new ObjectBaseToObjectBaseDTOMapper();
   _referenceMapper = new ObjectBaseDTOToReferenceNodeMapper(_objectBaseDTOMapper);
   
   otherwise, the Id`s of the objects are fake and I get this random string on them.
   
   If I alter this for the previous tests, they fail, since they are relying on this fake implementation as mocks.
   
   So I figured out this should be a different class. 
   
   let me know your thoughts about it. Maybe I could take this to a different file?